README.md

```diff
@@ -122,3 +122,24 @@ separately.
 - [Build Artifacts](docs/development/artifacts.md): Documentation about the outputs of the build system.
 - [Releases Page](RELEASES.md): Documentation for how to leverage our build artifacts.
 - [Roadmap for Support](ROADMAP.md): Documentation for our prioritized roadmap to support AMD GPUs.
+
+## Provisioning TheRock 🪨
+
+In order to provision TheRock using either a release tag or CI runner build, use the `build_tool/provision.py` script.
```

Comment:
```suggestion
In order to provision TheRock using either the latest developer or automated nightly release or a specific CI runner build, use the `build_tool/provision.py` script.
```

Maybe we should rephrase a little to make clear what a user gets as also the developer and nightly builds are CI runner builds.